lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -140,7 +141,7 @@ changePage = (doc, options) ->
     nodesToChange = [currentBody]
 
   triggerEvent EVENTS.BEFORE_UNLOAD, nodesToChange
-  document.title = title
+  document.title = title if title != false
```

Comment:
I'd prefer CoffeeScript's equality sugar here. `document.title = title if title isnt false` or `document.title = title unless title is false`